<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Base Converter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="styleSheet">
       <string notr="true">font: bold 20pt; </string>
      </property>
      <property name="text">
       <string>进 制 转 换 器</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputNumber">
      <property name="placeholderText">
       <string>Enter number</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLineEdit" name="fromBase">
        <property name="placeholderText">
         <string>From Base</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="swapButton">
        <property name="text">
         <string>⇆</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="toBase">
        <property name="placeholderText">
         <string>To Base</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="convertButton">
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="result">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Result</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
